File: python/linear_search.py
def linear(word) :
  arrayToSearchThrough = [];
  for l in word:
    arrayToSearchThrough.append(l);
  
  return arrayToSearchThrough;


banana_phone = linear("banana");

print(banana_phone)

def linear_search(value_to_find, array_to_search_through):
    arr = []
    for i, l in enumerate(array_to_search_through):
        if l == value_to_find:
            arr.append(i)
    return arr
    # your code here

print(linear_search('a', banana_phone))

File: js/linearSearch.js
function linear(word) {
  let arrayToSearchThrough = [];
  for (let i = 0; i < word.length; i++) {
    arrayToSearchThrough.push(word[i]);
  }
  return arrayToSearchThrough;
}

bananaPhone = linear("banana");

// console.log(bananaPhone);

// exports.linearSearch = function(valueToFind, arrayToSearchThrough) {

// };

const linearSearch = (item, arrayToSearchThrough) => {
  let arr = [];
  for (i = 0; i < arrayToSearchThrough.length; i++) {
    if (arrayToSearchThrough[i] === item) {
      arr.push(i);
    }
  }
  return arr;
};

console.log(linearSearch("a", bananaPhone));
